### DISCUSSION TITLE
Mesh Combiner Python Script for Multi-zone Meshes

### DISCUSSION THREAD

**Discussion by MKursatUzuner** [su2code/SU2#1506] [Show and tell]
Hi everyone,

I have made a simple python script for combining multiple native SU2 format meshes into one multi-zone mesh file. 

The script can be found in the link below. Hope it will be of help.

https://github.com/MKursatUzuner/Multi-zone-Mesh-Combiner-for-SU2

**Comment by TobiKattmann** [comment]
Hi, thanks for sharing :)

I have to admit I am not a 100% sure whether an additional NDIME=2|3 at the top of the file is necessary but I included that in my bash-script [here](https://github.com/TobiKattmann/Meshes/blob/master/combineSU2meshes.sh) ... Looking at some other files that might unnecessary precaution or it was needed back in the day. But if your script works fine then :+1:

**Reply by MKursatUzuner** [reply]
Hello,

I mainly used this script for 3D geometries and it didn't need NDIME for those ones. I tried on a very simple 2D multi-zone mesh and it seems like it is working for 2D as well without NDIME addition. Though it is always better to check if the mesh is good and working properly before and after combining them :)

Thanks for your comments.
